README.md

```
@@ -0,0 +1,23 @@
+I have now adapted the code so that each group's members can be outputed into individual CSV files which are
+appropriately named as the corresponding groups.
+
+## Requirements:
+
+You're going to need to install Telethon package for Python, and configure it to suit the Telegram API. 
+Navigate over to https://my.telegram.org and using the API development tools configure your credentials. 
+For this you are going to ned an `api_id` and `api_hash` and your phone number. 
+Paste these values in the appropriate fields in `member_go.py`.
+
+To install Telethon you can do:
+`python pip install telethon `
```

markdownlint-cli2 (0.17.2)

Spaces inside code span elements
null

(MD038, no-space-in-code)